utilities/pipelines/manageIssueForFailingPipelines/New-IssuesForFailingPipelines.ps1

```diff
@@ -0,0 +1,114 @@
+﻿<#
+.SYNOPSIS
+Check for failing pipelines and create issues for those, that are failing.
+
+.DESCRIPTION
+If a pipeline fails, a new issue will be created, with a link to the failed pipeline. If the issue is already existing, a comment will be added, if a new run failed (with the link for the new failed run). If a pipeline run succeeds and a issue is open for the failed run, it will be closed (and a link to the successful run is added to the issue).
+
+.PARAMETER repo
+Needs to have the structure "owner/repositioryName"
+
+.PARAMETER limitNumberOfRuns
+Optional. Number of recent runs to scan for failed runs. Default is 100.
+
+.PARAMETER limitInDays
+Optional. Only runs in the past selected days will be analyzed. Default is 2.
+
+.EXAMPLE
+New-IssuesForFailingPipelines -repo 'owner/repo01' -limitNumberOfRuns 100 -limitInDays 2 -Verbose -WhatIf
+
```

Comment:
Description of the above command is missing. You should add at least once sentence describing what this variant of the invocation would do

Reply:
Done